--- go
+++ rust
@@ -23 +23 @@
-ERR (1064, 'You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 18 near "-1) OVER (ORDER BY d) FROM w2" ') | WARN Error:1064:You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 18 near "-1) OVER (ORDER BY d) FROM w2" 
+ERR (1210, 'Incorrect arguments to lag') | WARN Error:1210:Incorrect arguments to lag
@@ -31 +31 @@
-ERR (1305, 'FUNCTION fdq.nth_value does not exist') | WARN Error:1305:FUNCTION fdq.nth_value does not exist
+ERR (1064, 'You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 30 near "FROM w2" ')
